engine/platform/sdl/in_sdl.c

```diff
@@ -25,6 +25,7 @@ GNU General Public License for more details.
 #include "vid_common.h"
 
 SDL_Joystick *g_joy = NULL;
+static SDL_Cursor *g_pDefaultCursor[CursorType_Last];
```

Comment:
Although SDL_Cursor is available in legacy SDL 1.2, it doesn't allow to load system cursors.

Let's hide it for 1.2 instead. InitCursors only works for 2.0 anyway.